setup.py

```diff
@@ -133,13 +144,24 @@ def cuda_info(cmd):
             return [incdir]
 
     if cmd == "libdirs":
-        for libdir in ("lib64", "lib"):
+        dirs = []
+        for libdir in ("lib64", "lib", "bin"):
             full_dir = os.path.join(cuda_path, libdir)
             if os.path.isdir(full_dir):
-                return [full_dir]
+                dirs.append(full_dir)
+        return dirs
 
     if cmd == "libs":
-        return ["cudart"]
+        if sys.platform.startswith("win32"):
+            pattern = os.path.join(cuda_path, "bin", "cudart64_*.dll")
```

Comment:
What are the CUDA RT libraries called?

Reply:
On Windows it's `C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.6\bin\cudart64_110.dll`, and on Linux it's `/usr/local/cuda-11.6/targets/x86_64-linux/lib/libcudart.so` (on my machine).

Reply:
I think we should do a similar existence check on Linux, just to be consistent. I.e., raise an error if the expected library isn't found.